lib/zluxArgs.js

```diff
@@ -107,6 +108,16 @@ if(process.env.overrideFileConfig !== "false"){
     }
     configJSON = mergeUtils.deepAssign(configJSON, envConfig);
   }
+  let zoweDotYamlConfig;
+  const zoweDotYamlFile = yamlConfig.getDefaultZoweDotYamlFile();
+  const haInstanceId = yamlConfig.getCurrentHaInstanceId();
+  if (zoweDotYamlFile && haInstanceId) {
+    zoweDotYamlConfig = yamlConfig.getConfig(zoweDotYamlFile, haInstanceId);
```

Comment:
This still needs component names to be specified.
Which components should be merged? Default which you coded is [ app-server, zss ] but forks and future enhancements may want more or different components loaded into `zoweDotYamlConfig`.

Reply:
Done